Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -117,6 +118,7 @@ extension LoginViewController: EndpointDelegate{
             //Notify user of unauthorized reload the view on the main thread
             DispatchQueue.main.async {
                 self.reloadViewWithUsernameAndInstanceSaved()
+                self.viewDidLoad()
```

Comment:
Also the done button is not disappearing when the "One or more required fields is missing" error occurs. Can anyone else verify this? And do we want the keyboard to go away if some fields are empty?